blockly-core/core/ui/contract_editor/contract_editor.js

```diff
@@ -823,6 +823,15 @@ Blockly.ContractEditor.prototype.resetExample = function (block) {
   this.testResetHandler_(block);
 };
 
+Blockly.ContractEditor.prototype.updateExampleResult = function (block, result) {
+  this.exampleViews_.forEach(function (view) {
+    if (view.isViewForBlock(block)) {
+      view.setResult(result);
+    }
+    return;
```

Comment:
Since it's in the `forEach` this won't break early, could use `.some` and return `true` when found or `.filter` for isView and then `.forEach` perform the operation on the (hopefully only) result